understanding/20/language-of-page.html

```diff
@@ -8,6 +8,15 @@
 <body>
    <h1>Understanding Language of Page</h1>
    
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Users can identify the language of a page</dd>
```

Comment:
@gundulaniemann said
> In fact it is about the language being programmaticaly determined, so for example screen readers pronounce understandably.
The user cannot even see the indication.

```suggestion
		<dt>Objective</dt><dd>Assistive technology can determine the language of a page</dd>
```

Reply:
fine with me